python/bindings/openravepy_int.cpp

```diff
@@ -244,7 +244,14 @@ void toRapidJSONValue(const object &obj, rapidjson::Value &value, rapidjson::Doc
 #endif
     else if (PyLong_Check(obj.ptr()))
     {
-        value.SetInt64(PyLong_AsLong(obj.ptr()));
+        int overflow;
+        const long result = PyLong_AsLongAndOverflow(obj.ptr(), &overflow);
+        if (overflow) {
+            value.SetUint64(result);
```

Comment:
```suggestion
            value.SetUint64(PyLong_AsUnsignedLong(obj.ptr()));
```

Reply:
Thank you, I also reached the same idea.